Gemfile

```diff
@@ -6,6 +6,7 @@ gem 'rufus-scheduler'
 gem 'sinatra'
 gem 'sinatra-contrib'
 gem 'thin'
+gem 'eventmachine'
```

Comment:
This gem wasn't before? 🤔 😄

Reply:
I mean do we have to add this gem? It wasn't there before. It is a dependency of Thin so it is added automatically, we don't need to add it explicitly.